yarn.lock

```diff
@@ -15653,6 +15726,20 @@ viem@^1.1.4:
     isomorphic-ws "5.0.0"
     ws "8.13.0"
 
+viem@^2.7.9:
```

Comment:
I have a feeling we were already using viem as a dep of some other pkg so if possible would be great to match versions and avoid dupes

Reply:
makes sense, i tried to bump it quickly but got some conflicts, will take a look again

Reply:
just did it 👌